app/src/main/java/org/p2p/wallet/home/ui/main/empty/BigBannerViewHolder.kt

```diff
@@ -0,0 +1,56 @@
+package org.p2p.wallet.home.ui.main.empty
+
+import android.graphics.drawable.ColorDrawable
+import android.graphics.drawable.Drawable
+import android.graphics.drawable.GradientDrawable
+import android.graphics.drawable.ShapeDrawable
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.annotation.ColorRes
+import androidx.core.content.ContextCompat
+import androidx.recyclerview.widget.RecyclerView
+import org.p2p.wallet.databinding.ItemBigBannerBinding
+import org.p2p.wallet.home.model.HomeBanner
+
+class BigBannerViewHolder(
+    private val binding: ItemBigBannerBinding,
+    private val onBannerButtonClicked: (buttonId: Int) -> Unit,
+) : RecyclerView.ViewHolder(binding.root) {
+
+    constructor(
+        parent: ViewGroup,
+        onBannerButtonClicked: (buttonId: Int) -> Unit
+    ) : this(
+        binding = ItemBigBannerBinding.inflate(LayoutInflater.from(parent.context), parent, false),
+        onBannerButtonClicked = onBannerButtonClicked
+    )
+
+    fun onBind(item: HomeBanner) = with(binding) {
```

Comment:
1. maybe we can put click listener inside HomeBanner too?
2. HomeBanner -> HomeBannerItem?